core/server/api_container/server/startosis_engine/startosis_interpreter.go

```diff
@@ -386,6 +391,16 @@ func (interpreter *StartosisInterpreter) buildBindings(
 	return &predeclared, nil
 }
 
+const (
+	moduleIdSeperator     = "/"
+	numModuleIdSeparators = 4
+	prefixNum             = 3
+)
+
+func getModulePrefix(moduleId string) string {
```

Comment:
Reviewing this, it'd have been helpful to have a comment giving an example of `getModulePrefix` is supposed to return so that I know how to model its output in the above code